code/modules/medical/cloning.dm

```diff
@@ -429,6 +429,25 @@
 	occupant.forceMove(exit)
 	icon_state = "pod_0"
 	eject_wait = FALSE //If it's still set somehow.
+	//do early ejection damage
+	var/completion = 10*((occupant.health + 100) / (heal_level + 100)) //same way completion is calculated for examine text, but out of 10 instead of 100
+	var/damage_rolls = 10 - round(completion) - (round(resource_efficiency) - 1) // 1 roll for each 10% missing, each improved pair of manipulators reduces one roll
+	var/hits = 0
+	while(damage_rolls > 0)
+		if(prob(25))//each roll has a 25% chance to give the occupant a bad time
+			hits++
+		damage_rolls--
+	//apply the damage
+	var/mob/living/carbon/human/H = occupant
+	while(hits>0)
+		if (hits>=4)
+			H.organs_by_name[pick(H.organs_by_name)].droplimb(override = 1, no_explode = 1, spawn_limb = 0, display_message = FALSE)
+			hits -= 4
+		else
+			pick(H.internal_organs).Destroy()
```

Comment:
```suggestion
			qdel(pick(H.internal_organs))
```

Reply:
/datum/organs/internal overwrites Destroy(), is it still okay to swap it out with qdel?

Reply:
qdel ends up calling Destroy, you're not supposed to call Destroy directly even though currently it does almost the same thing.

Reply:
OK, will resolve in next commit to this branch